test/common/default_email.go

```diff
@@ -66,8 +88,20 @@ func TestDefaultUserEmail(t TestingTB, ctx *TestingContext) {
 		return
 	}
 
+	keycloakUser3, err := waitForKeycloakUser(ctx, 5*time.Minute, rhssoNamespace, usernameUsingEmail)
+	if err != nil {
+		t.Fatalf("Unexpected error querying KeycloakUser %s: %v", usernameUsingEmail, err)
+		return
+	}
+
+	keycloakUser4, err := waitForKeycloakUser(ctx, 5*time.Minute, rhssoNamespace, usernameRequireSanitiseEmail)
+	if err != nil {
+		t.Fatalf("Unexpected error querying KeycloakUser %s: %v", usernameUsingEmail, err)
```

Comment:
```suggestion
		t.Fatalf("Unexpected error querying KeycloakUser %s: %v", usernameRequireSanitiseEmail, err)
```
Copy / Paste typo. Going to push a fix for this :man_technologist: